client/jetpack-connect/persistence-utils.js

```diff
@@ -24,3 +30,31 @@ export const retrievePlan = () => {
 	const cookies = cookie.parse( document.cookie );
 	return cookies.jetpack_connect_selected_plan;
 };
+
+export const persistSession = url => {
+	const options = {
+		maxAge: JETPACK_CONNECT_TTL_SECONDS,
+		path: '/',
+	};
+	document.cookie = cookie.serialize( 'jetpack_connect_session_url', urlToSlug( url ), options );
+};
+
+export const isCalypsoStartedConnection = siteSlug => {
+	const cookies = cookie.parse( document.cookie );
+	return cookies.jetpack_connect_session_url === urlToSlug( siteSlug );
+};
+
+export const persistFlowType = flowType => {
+	const options = { path: '/' };
+	document.cookie = cookie.serialize( 'jetpack_connect_session_flowtype', flowType, options );
```

Comment:
Maybe we should only allow certain values to be written here?

Reply:
👍  `flowTypes`:

https://github.com/Automattic/wp-calypso/blob/fc5b6a060e7c0eeb4c2871301a65a96043677c2c/client/jetpack-connect/main.jsx#L52